avd-containers/base/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG FROM_IMAGE
+ARG FROM_VARIANT
+FROM ${FROM_IMAGE}:${FROM_VARIANT}
+
+# install essential tools
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    make \
+    git \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -Rf /usr/share/doc && rm -Rf /usr/share/man \
+    && apt-get clean
+
+# add entrypoint script
+COPY ./entrypoint.sh /bin/entrypoint.sh
+RUN chmod +x /bin/entrypoint.sh
+# use ENTRYPOINT instead of CMD to ensure that entryscript is always executed
```

Comment:
entrypoint could be overwritten so there is no guarantee here - I don't think the comment is useful though

Reply:
I'm happy to rewrite the comment. This is for future manual run of dev container. With VSCode entrypoint is always overwritten. If you have better suggestions - happy to change, otherwise I'll keep it as a reminder for myself. It's hard to explain the whole story in Dockerfile, but it triggers some memories in my head. =)